builders/integrations/bridges/axelar.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Going Cross-Chain Via Axelar Network
+description: Learn how to bridge assets, plus other ways you can connect your Moonbeam dApp to assets and functions on multiple blockchains using Axelar network & APIs.
+---
+
+# Axelar Network
+
+## Introduction 
+
+Axelar delivers secure cross-chain communication for Web3. Our infrastructure enables dApp users to interact with any asset or application, on any connected chain, with one click. Powered by a permissionless Proof-of-Stake validator set, Axelar network allows dApps to pass arbitrary messages across chains using Turing-complete calls.
```

Comment:
A link back to your website would be helpful at first mention. Also, try to avoid using "our" instead say "Axelar's" or something like that